Tutorials/01_Psi4NumPy-Basics/1d_wavefunction.ipynb

```diff
@@ -229,7 +229,7 @@
     "|----------|-------------|-------------|\n",
     "| Orbital Coefficients, **C** | [wfn.Ca()](http://psicode.org/psi4manual/master/psi4api.html#psi4.core.Wavefunction.Ca \"Go to Documentation\"), [wfn.Cb()](http://psicode.org/psi4manual/master/psi4api.html#psi4.core.Wavefunction.Cb \"Go to Documentation\") | Returns orbital coefficient matrix for $\\alpha$ (Ca) or $\\beta$ (Cb) orbitals. (Identical for restricted orbitals) |\n",
     "| Electron Density, **D** | [wfn.Da()](http://psicode.org/psi4manual/master/psi4api.html#psi4.core.Wavefunction.Da \"Go to Documentation\"), [wfn.Db()](http://psicode.org/psi4manual/master/psi4api.html#psi4.core.Wavefunction.Db \"Go to Documentation\") | One-particle density matrices for $\\alpha$ (Da) and $\\beta$ (Db) electrons. (Identical for restricted orbitals) |\n",
-    "| Fock Matrix, **F** | [wfn.Fa()](http://psicode.org/psi4manual/master/psi4api.html#psi4.core.Wavefunction.Fa \"Go to Documentation\"), [wfn.Fa()](http://psicode.org/psi4manual/master/psi4api.html#psi4.core.Wavefunction.Fb \"Go to Documentation\") | Returns the Fock matrix. For wavefunction with unrestricted orbitals, distinct Fock matrices $F^{\\alpha}$ and $F^{\\beta}$ for $\\alpha$ and $\\beta$ orbitals, respectively, are created.|\n",
+    "| Fock Matrix, **F** | [wfn.Fa()](http://psicode.org/psi4manual/master/psi4api.html#psi4.core.Wavefunction.Fa \"Go to Documentation\"), [wfn.Fb()](http://psicode.org/psi4manual/master/psi4api.html#psi4.core.Wavefunction.Fb \"Go to Documentation\") | Returns the Fock matrix. For wavefunction with unrestricted orbitals, distinct Fock matrices $F^{\\alpha}$ and $F^{\\beta}$ for $\\alpha$ and $\\beta$ orbitals, respectively, are created.|\n",
```

Comment:
Shouldn't this actually be changed to `Fa` across the board?

Reply:
Since the link led to documentation for psi4.core.Wavefunction.Fb, I figured that it should be changed to wfn.Fb() in the table as well, especially since the description discusses the distinct alpha and beta Fock matrices for the unrestricted case. Similarly, the table lists wfn.Ca() and wfn.Cb() as well as wfn.Da() and wfn.Db() directly above. But maybe I misunderstood.

Reply:
Works for me.